lib/next_ls/runtime/sidecar.ex

```diff
@@ -15,11 +15,34 @@ defmodule NextLS.Runtime.Sidecar do
   end
 
   def handle_info({:tracer, payload}, state) do
+    module_name = payload.module |> to_string() |> String.replace("Elixir.", "")
```

Comment:
You can use `Macro.to_string/1` instead of chopping off the `Elixir.` prefix

Reply:
simplified that to ditch strings here